src/test/java/org/apache/commons/collections4/multimap/UnmodifiableMultiValuedMapTest.java

```diff
@@ -100,6 +96,68 @@ public void testAddException() {
         }
     }
 
+    public void testRemoveException() {
+        final MultiValuedMap<K, V> map = makeFullMap();
+        try {
+            map.remove((K) "one");
+            fail();
+        } catch (final UnsupportedOperationException e) {
+        }
```

Comment:
Comment empty blocks so we know why we are ignoring exceptions. It helps others to understand the code more quicly IMO.